### PR TITLE
Feature: Multiple configs input.

### PR DESCRIPTION
**Pull request by h-alice**
We implemented a way to accept multiple profiles input, inspired from `docker-compose`.
Now we can use the tool like:
```
imagetool -f b5-size.yaml -f a4-size.yaml -f postcard.yaml input-image1.png input-image2.jpg
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 *.exe
 vendor
 .DS_Store
+
+# Local IDE generated files.
+.vscode
```
Comment (NatLee) [inline .gitignore:6]: desktop.ini may be in this ignore file 🥹🥹🥹

File: main.go [hunk 2]
```diff
@@ -36,42 +49,55 @@ func ProcessFile(profile config.ProcessProfileConfig, in io.Reader) error {
...
+			loaded_configs = append(loaded_configs, conf) // Append to loaded configs.
+		}
+		config_root = config.MergeConfigFiles(loaded_configs...) // Merge all loaded configs.
+	} else { // If path not specified, load defaultprofile from home directory.
 
 		log.Printf("[!] Using default config file.")
 
-		var err error
-		config_path, err = getProfileFromHomeDir("default", true)
+		config_path, err := getProfileFromHomeDir("default", true)
```
Comment (NatLee) [inline main.go:75]: := 

🤩🤩🤩

**Review by NatLee — APPROVED** [review]
I think this is good to go!
🥳🥳🥳

File: .gitignore (hunk 1 above)
Comment (h-alice) [reply .gitignore:6]: desktop.ini is so bad 🥺

File: main.go (hunk 2 above)
Comment (h-alice) [reply main.go:75]: One should use ':=' iff declaring new variable.
That's a pitfall. 🕳️

**Comment by h-alice** [comment]
Let's GOOOOOOOOOOO

**Pull request merged**